external/docs/docs.depproj

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier>none</NuGetRuntimeIdentifier>
+    <OutputPath>$(XmlDocDir)</OutputPath>
+    <IsRuntimeAssembly>false</IsRuntimeAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="$(XmlDocPackage)">
+      <Version>$(XmlDocPackageVersion)</Version>
+    </PackageReference>
+  </ItemGroup>
+
+  <!-- the intellisense package doesn't use nuget conventions so we need to select manually -->
+  <Target Name="GetDocFiles" AfterTargets="ResolveReferences">
+    <ItemGroup>
+      <DocFile Include="$(XmlDocFileRoot)/**/*.xml"/>
+      <DocFile>
+        <!-- trim off slash since it differs by platform and we need to do a string compare -->
+        <LCID>$([System.String]::new('%(RecursiveDir)').TrimEnd('\').TrimEnd('/'))</LCID>
```

Comment:
Sucks that there isn't any prettier way to do this manipulation and that we repeated it a couple times.